src/workloads/docs/Generators.yml

```diff
@@ -332,6 +335,28 @@ Actors:
             # Maps are identified by their "id" are shared.
             freqmap_singleton: { ^TakeRandomStringFromFrequencyMapSingleton: { id: "some_string", from: { "string3":100, "string4":200 } } }
 
+            # UUID
+            uuids: [
+              # Short uuid
+              {^UUID: {hex: {^RandomString: {length: 8, alphabet: "0123456789ABCDEF"}}}},
```

Comment:
All of your tests above are for full length. I was confused when I got here that this was even possible. Could you add a short uuid passing test a well if this is a thing? 

I went to the wikipedia page and got even more confused, so I'm going to rely on your answer on whether short UUIDs are a thing.